CMakeLists.txt

```diff
@@ -102,6 +99,36 @@ else()
   set(USE_UPDATER NO)
 endif()
 
+# Enable the inclusion of fonts currently carried in the source code by default
+# unless the environmental variable WITH_FONTS is defined and is not set to
```

Comment:
Shouldn't this read `[...]is defined and is set to[...]`?

Reply:
Ah, yeah a double negative! Will touch that up...